cmake/FindBenchmark.cmake

```diff
@@ -0,0 +1,20 @@
+# Distributed under the MIT License.
```

Comment:
Did you write this file or get it from somewhere?  If you wrote it, I prefer FindGoogleBenchmark.cmake; if you didn't we should include the license from where you got it...

Reply:
Yeah I'll change the name to be consistent.

Reply:
This cannot work. You didn't rename this file to `FindGoogleBenchmark`